mmpose/datasets/datasets/body3d/body3d_base_dataset.py

```diff
@@ -60,6 +60,9 @@ def load_config(self, data_cfg):
         self.seq_len = data_cfg.get('seq_len', 1)
         self.seq_frame_interval = data_cfg.get('seq_frame_interval', 1)
         self.causal = data_cfg.get('causal', True)
+        self.pad = data_cfg.get('pad', False)
```

Comment:
Is this temporal padding? Maybe we can use a more specific term like "temporal_padding".

Reply:
We may need to describe the options in data_cfg in the docstring.